modules/ROOT/pages/using-the-library.adoc

```diff
@@ -0,0 +1,40 @@
+= Using the library
```

Comment:
Could this be a bit confusing title?

I expected this to describe how to use the library, not how to integrate with my stack